src/pybind/mgr/cephadm/module.py

```diff
@@ -3173,6 +3174,16 @@ def set_custom_prometheus_alerts(self, alerts_file: str) -> str:
 
     @handle_orch_error
     def set_prometheus_target(self, url: str) -> str:
+        try:
+            if url.startswith("http://") or url.startswith("https://"):
+                return f"Invalid URL '{url}'. It should be in the format host_ip:port"
```

Comment:
This error is confusing!! we are telling the user that the input is not a valid "URL" when in fact what we are asking for is an `ip:port` !! we should be more concise. I think we can say something like: 

``` f"Invalid input '{url}'. Expected format: 'host_ip:port' (e.g., '127.0.0.1:8080'). Ensure the URL does not include 'http://' or 'https://'." ```

Reply:
Updated, Thanks